integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -142,7 +142,7 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
           fileInfo.getPersistenceState());
 
       // sleep and wait for worker to persist the file
-      CommonUtils.sleepMs(5);
+      CommonUtils.sleepMs(200);
```

Comment:
The persistence is done in a thread, and takes time. Any other way is to keep polling the worker via heartbeat until it's done. And the test will timeout if it never finishes. What do you think?